Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -889,84 +893,92 @@ - (instancetype)initWithParentAdapter:(ALVungleMediationAdapter *)parentAdapter
         self.adFormat = adFormat;
         self.parameters = parameters;
         self.delegate = delegate;
+        self.isAdloadSuccess = NO;
     }
     return self;
 }
 
-- (void)bannerAdDidLoad:(VungleBanner *)banner
+- (void)bannerAdDidLoad:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView loaded: %@", banner.placementId];
-    
-    NSString *creativeIdentifier = banner.creativeId;
+    [self.parentAdapter log: @"AdView loaded: %@", bannerView.placementId];
+
+    self.isAdloadSuccess = YES;
+    NSMutableDictionary *extraInfo = [NSMutableDictionary dictionaryWithCapacity: 3];
+    BOOL isExtraInfo = NO;
```

Comment:
i think MAX will comment on this. 
so i think u can just do what google dose. 
https://github.com/AppLovin/AppLovin-MAX-SDK-iOS/blob/master/Google/GoogleAdapter/ALGoogleAdViewDelegate.m#L37

Check for ALSdk.versionCode >= 6150000 for if else. Thanks @ashinagawa

Reply:
Updated.